cmd/translatesfx/translatesfx/otel.go

```diff
@@ -410,7 +417,7 @@ func dimsToMetricsTransformProcessor(m map[interface{}]interface{}) map[string]i
 func receiverList(receivers map[string]map[string]interface{}) []string {
 	var keys []string
 	for k := range receivers {
-		if k == processlist {
```

Comment:
I wonder if it would make things more understandable to readers if `k == processlist || k == kubernetesEvents` were factored into an `isLogsReceiver` function.

Reply:
I tend toward the "third time's a charm" approach for breaking out helpers but makes sense either way.

When I was updating this it became apparent the receiver creator logic isn't monitor type-dependent, which could be problematic.  It's not immediately clear to me when discovery rules would be used for sfx forward, processlist, or k8s events monitors so I didn't attempt to resolve it in these changes but did make the `receiverList` function more specific.  This ok?

Reply:
Ok no worries -- I wasn't really thinking of this as a helper but rather as a way to document by breaking this into its own function with a descriptive name.

I'm not quite sure I understand the receiver creator comment -- but basically discovery rules for sfx forwarder, processlist, or k8s events are not supported (at least not yet). And `receiverList` should be renamed to `metricsReceiverList` (this code could use a cleanup). We can discuss separately if you want, but your change looks fine to me.